state-chain/custom-rpc/src/lib.rs

```diff
@@ -487,6 +489,9 @@ pub trait CustomApi {
 		&self,
 		broadcast_id: BroadcastId,
 	) -> RpcResult<Option<<cf_chains::Ethereum as Chain>::Transaction>>;
+
+	#[method(name = "witness_count")]
+	fn cf_witness_count(&self, hash: String) -> RpcResult<Option<FailingWitnessValidators>>;
```

Comment:
I think you can use `state_chain_runtime::Hash` as the type here and it will deserialize correctly. Avoids having to use `from_hex` to convert the string.